src/Kunstmaan/AdminBundle/Controller/ExceptionController.php

```diff
@@ -29,8 +29,53 @@ public function indexAction(Request $request)
         return parent::doIndexAction($this->getAdminListConfigurator(), $request);
     }
 
+    /**
+     * @Route("/resolve_all", name="kunstmaanadminbundle_admin_exception_resolve_all")
+     *
+     * @return RedirectResponse
+     *
+     * @throws \Doctrine\ORM\ORMInvalidArgumentException
+     * @throws \Doctrine\ORM\OptimisticLockException
+     * @throws \InvalidArgumentException
+     */
+    public function resolveAllAction()
+    {
+        /* @var EntityManager $em */
+        $em = $this->getEntityManager();
+
+        $this->getAdminListConfigurator();
+
+        $exceptions = $em->getRepository(Exception::class)->findAllNotResolved();
```

Comment:
It's maybe better to create an update query for this. Right now you are loading all Exception entities inside the entitymanager.